app/src/components/recent-searches.tsx

```diff
@@ -0,0 +1,122 @@
+import { Box, Text } from "@chakra-ui/react";
+import React from "react";
+
+const RecentSearches = () => {
+  const hasRecentSearches = true;
+  return (
+    <Box>
+      <Box className="px-4">
+        <Text
+          color="content.tertiary"
+          fontWeight="extrabold"
+          lineHeight="16px"
+          letterSpacing="widest"
+          fontSize="overline"
+          fontFamily="heading"
+        >
+          RECENT SEARCHES
+        </Text>
+      </Box>
+      {hasRecentSearches ? (
+        <Box>
+          <ul className="list-none">
+            <li>
+              <Box className="h-[72px] w-full flex flex-col justify-center group px-4 hover:bg-[#2351DC] transition-all duration-150 cursor-pointer">
+                <Text
+                  className="group-hover:text-white"
+                  color="content.secondary"
+                  fontSize="body.lg"
+                  fontFamily="body"
+                  fontWeight="normal"
+                  lineHeight="24"
+                  letterSpacing="wide"
+                >
+                  Rio de Janeiro
+                </Text>
+                <Text
+                  className="group-hover:text-[#E8EAFB]"
```

Comment:
You can add custom colors to the Tailwind config.
In this case better would be the following Chakra attribute, which allows you to use theme colors:
`_hover={{ color: "color.name" }}`